save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/BanView.kt

```diff
@@ -0,0 +1,67 @@
+/**
+ * View for Ban
+ */
+
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.frontend.components.views
+
+import com.saveourtool.save.frontend.utils.Style
+import com.saveourtool.save.frontend.utils.useBackground
+import com.saveourtool.save.info.UserInfo
+import com.saveourtool.save.info.UserStatus
+import js.core.jso
+import react.FC
+import react.PropsWithChildren
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.p
+import react.router.useNavigate
+import web.cssom.ClassName
+import web.cssom.rem
+
+private const val BAN = """
+    BAN
+"""
+
+private const val SUPPORT = """
+    You have received a ban on our platform. If you think that this happened by mistake, please contact our support.
```

Comment:
You have been banned from our platform. If you believe this was a mistake, please contact our support team at saveourtool@gmail.com.

Reply:
Fixed